index.js

```diff
@@ -75,7 +75,10 @@ class Machine {
 
     for (const key in options) {
       if (hasOwnProperty.call(options, key)) {
-        args.push(`--${key}`, options[key])
+        args.push(`--${key}`)
+        if (options[key]) {
+          args.push(options[key])
+        }
```

Comment:
How about:

```js
for (const key in options) {
  if (hasOwnProperty.call(options, key)) {
    if (key === 'args' && Array.isArray(options[key])) {
      args.push(...options[key])
    } else {
      // TODO (future major): remove in favor of args option
      args.push(`--${key}`, options[key])
    }
  }
}
```

Reply:
I don't know what will be easier for end users :). As for me both options are good.

Reply:
Let's go with this, because I like that these `args` can be passed to `docker-machine` as-is.